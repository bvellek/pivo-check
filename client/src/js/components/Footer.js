import React from 'react';

const Footer = () => (
  <footer className="landing-footer" role="contentinfo">Designed and Developed by <a href="https://benvellek.com/" target="_ben-portfolio">BEN VELLEK</a></footer>
);

export default Footer;
