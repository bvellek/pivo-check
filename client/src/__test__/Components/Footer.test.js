import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Footer from '../../js/components/Footer';

describe('<Footer />', () => {
  it('renders without crashing', () => {
    shallow(<Footer />);
  });
  it('it renders correctly', () => {
    const component = renderer.create(
      <Footer />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
