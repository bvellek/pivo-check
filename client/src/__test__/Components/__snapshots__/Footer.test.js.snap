// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> it renders correctly 1`] = `
<footer
  className="landing-footer"
  role="contentinfo"
>
  Designed and Developed by 
  <a
    href="https://benvellek.com/"
    target="_ben-portfolio"
  >
    BEN VELLEK
  </a>
</footer>
`;
